src/schema/sources.ts

```diff
@@ -1058,6 +1065,11 @@ export const resolvers: IResolvers<any, Context> = {
         filter.type = SourceType.Squad;
         filter.private = false;
       }
+
+      if (!isNullOrUndefined(args.featured)) {
+        filter.flags = { featured: args.featured };
+      }
```

Comment:
You are gonna have to do nested `andWhere` and specifiy `flags->'featured' = :featured` because like this typeorm creates a query that just compares the full jsonb field:
`AND WHERE "source_1fm"."flags" = :orm_param_1`, which would fail if you had any other jsonb properties in flags.

Reply:
and yes, typeorm jsonb support is garbage :hidethepain:

Reply:
Should now be converted to this. All tests are still passing 🚀